view/display.py

```diff
@@ -0,0 +1,117 @@
+                # y
+                fill='black')
+            tk.update()
+            self._state.floor_population[person.current_floor] += 1
+
+        self._animation_speed = animation_speed
+        self._canvas = canvas
+        self._tk = tk
+
+    def move_slowly(self, animation, x, y):
+        """This function animates people entering and exiting the elevator"""
+        for j in range(0, 50):
+            self._canvas.move(animation, x / 50, y / 50)
+            self._tk.update()
+            time.sleep(self._animation_speed / 500)
+
+    def arriving_person(self, person, car):
+        self._elevator_animation[person.elevator_spot] = False
+        self._canvas.itemconfig(person.animation, fill='green')
+        self._arrivals_population[car.current_floor] += 1
+        self.move_slowly(person.animation, 390 + self._arrivals_population[car.current_floor] * 12 -
+                         self._canvas.coords(person.animation)[0], 15 * (person.elevator_spot % 2))
+        self._canvas.itemconfig(self._delivered_label,
+                                text='Arrived - ' + str(self._state.status['arrived_people_number']))
+        self._canvas.itemconfig(self._inside_label,
+                                text='Inside elevator - ' + str(car.in_people))
+        self._canvas.itemconfig(self._waiting_label,
+                                text='Waiting - ' + str(
+                                    self._state.people_number - self._state.status['arrived_people_number']))
+
+    def in_car(self, person, car):
+        for spot in range(len(self._elevator_animation)):
+            if not self._elevator_animation[spot]:
+                self._elevator_animation[spot] = True
+                person.elevator_spot = spot
+                self.move_slowly(person.animation, (275 + (spot % 3) * 15) -
+                            self._canvas.coords(person.animation)[0], -15 * (spot % 2))
+                break
+        self._canvas.itemconfig(person.animation, fill='white')
+        self._canvas.itemconfig(self._inside_label,
+                          text='Inside elevator - ' + str(car.in_people))
+        self._canvas.itemconfig(self._waiting_label,
+                          text='Waiting - ' + str(
+                                    self._state.people_number - self._state.status['arrived_people_number']))
+
+    def iteraction(self):
+        for i in range(self._floor_height):
+            self._tk.update()
+            time.sleep(self._animation_speed / self._floor_height)
+            self._canvas.move(self._elevator, 0, -self._state.status['cars'][0].direction)  # animate the lift moving
+            for person in self._state.status['people']:
+                if person.in_car:
+                    self._canvas.move(person.animation, 0,
+                                -self._state.status['cars'][0].direction)  # animate the people moving
+        time.sleep(self._animation_speed)
+
+    def finish(self):
```

Comment:
I can see the stats are all printed, we would eventually generate a CSV file, we can do an issue for that

Reply:
OK issue #10 created for this.